core/resharding.go

```diff
@@ -171,42 +166,10 @@ func CalculateNewShardState(
 		return nil, ctxerror.New("cannot retrieve previous sharding state").
 			WithCause(err)
 	}
-	newNodeList := ss.UpdateShardingState(stakeInfo)
```

Comment:
Why do we remove these two line here? This func is not used but I don't see the need to remove these two line specifically now.

Reply:
Sorry, optimistic dead code slashing